packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -446,6 +357,23 @@ import {
 const workflowEventBus = workflowService.workflowEventBus;
 const WORKFLOW_SAVE_INTERVAL = 8000;
 
+// FIXME: check if there is a component typing instead of any
```

Comment:
I think `Component` is a type. Also I am not sure if using a map is ideal, I thought objects were faster when it comes to reading.

Reply:
I believe you can just pass a string to the `is` prop in `<component>` and it'll know what you mean. You can do something like `:is="`Tera${Operation.name}`"`. Maybe that loses strictness though

Reply:
Our component naming scheme isn't as uniform as you think, there are some slight variations that will fail.

Object access is faster, but barely noticeable until we get around to hundreds of thousands of accumulated accesses. Here is a mini-test, obj is a little over twice as fast in this somewhat trivial test. 200K access attempt, 7 milliseconds vs 3 milliseconds.

Result
```
obj 3
map 6.799999952316284
true
```

Test
```
const map = new Map();
const obj = {};
const limit = 80000
for (let i = 0; i < limit; i++) { map.set(i, i); obj[i] = i; }

let start = 0;
let end = 0;
let seq = [];
let objCount = 0;
let mapCount = 0;

for (let i = 0; i < 200000; i++) { seq.push(Math.round(Math.random() * limit)); }

start = performance.now();
for (let i = 0; i < seq.length; i++) {
  if (obj.hasOwnProperty(seq[i])) {
    objCount ++;
  }
}
end = performance.now();
console.log('obj', end - start)


start = performance.now();
for (let i = 0; i < seq.length; i++) {
  if (map.has(seq[i])) {
    mapCount ++;
  }
}
end = performance.now();
console.log('map', end - start)

console.log(objCount === mapCount);
```